packages/dd-trace/test/appsec/index.fastify.plugin.spec.js

```diff
@@ -536,6 +522,89 @@ withVersions('fastify', 'fastify', '>=2', version => {
       })
     })
   })
+
+  describe('Suspicious request blocking - multipart', () => {
+    withVersions('fastify', '@fastify/multipart', multipartVersion => {
+      let server, uploadSpy, axios
+
+      before(function () {
+        // @fastify/multipart is not compatible with Fastify 2.x
+        if (version.startsWith('2')) {
```

Comment:
version check with semver is more accurate
```suggestion
        if (semver.intersects(version, '<= 2.x')) {
```